src/modules/Invoice/Service.php

```diff
@@ -164,14 +164,17 @@ public function toApiArray(\Model_Invoice $invoice, $deep = true, $identity = nu
         }
         $tax = $tax_total;
 
+        $invoice_number_zero_padding = $this->di['mod_service']('system')->getParamValue('invoice_number_zero_padding');
+        $invoice_number_zero_padding = !empty($invoice_number_zero_padding) ? $invoice_number_zero_padding : 5;
```

Comment:
It's worth mentioning that in the unlikely case someone wants to have zero padding, the usage of `empty` would prevent that and instead result in 5 being used.

Test code:
```PHP
if(empty('0')){
	echo "0 is empty";
} else {
	echo "0 is not empty";
}
```

Gives:
> 0 is empty

Although not as pretty & readable, this would ensure people can specify zero padding if they desired:
```PHP
$invoiceNumberPadding = $invoiceNumberPadding !== null && $invoiceNumberPadding !== '' ? $invoiceNumberPadding : 5;
```

Reply:
Resolved https://github.com/devadattas/FOSSBilling/commit/d38f037320d1e224c449db3fcdc3f059f3571b5a